app/main.py

```diff
@@ -1,8 +1,35 @@
 class Car:
-    # write your code here
-    pass
+    def __init__(self, klass, clean, brand):
+        self.comfort_class = klass
+        self.clean_mark = clean
+        self.brand = brand
 
 
 class CarWashStation:
-    # write your code here
-    pass
+    def __init__(self, distance, clean, rating, count):
```

Comment:
better use more descriptive names for the parameters, because the person who will use your class may not understand what he has to pass to it.